test/system.jl

```diff
@@ -190,6 +190,9 @@
 
             @testset "SystemDA only accessors" begin
                 @test get_initial_commitment(da_system) isa KeyedArray{Bool, 1}
+                @test get_initial_uptime(da_system) isa KeyedArray{Float64, 1}
```

Comment:
can we test the values too to ensuree we got the logic correct? (e.g. that we haven't flipped downtime and uptime)